.circleci/config.yml

```diff
@@ -534,3 +554,6 @@ workflows:
       - docs-deploy:
           xcode_version: '14.0.0'
           <<: *release-tags
+  danger:
+    jobs:
+      - danger
```

Comment:
Do we need to specify in what branches we need to run this? I guess we want every branch that has a PR associated? (Though not sure how to do that in circleci)

Reply:
With the settings we have in CircleCI for purchases-ios it will work like that and this will run in every branch with a PR

<img width="1014" alt="Screen Shot 2022-08-05 at 2 02 35 PM" src="https://user-images.githubusercontent.com/664544/183073461-f2e3ef5e-60ca-4673-9057-13ea959ab753.png">

Reply:
Oh interesting... I don't think that's the case for all our repos, which might lead to inconsistencies. But it should be ok in this repo then 👍